demos/graspTheBall/main.yml

```diff
@@ -16,7 +16,7 @@ x-yarp-base: &yarp-base
     - "/tmp/.X11-unix:/tmp/.X11-unix"
     - "${XAUTHORITY}:/root/.Xauthority"
     - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
-    - "${APPSAWAY_DEMOREDBALL_CONTEXT}:/usr/local/share/ICUBcontrib/contexts/demoRedBall/:rw"
+    - "${APPSAWAY_DEMOREDBALL_CONTEXT}:/usr/local/share/ICUBcontrib/contexts/demoRedBall/"
```

Comment:
Doesn't this demo generate a file from the color calibration? is it used only inside the containers?

Reply:
It does not generate new files, but only modifies the ones on the host. So we don't need the `rw`.